src/brightness/ddcutil.rs

```diff
@@ -78,7 +78,17 @@ fn find_display_by_name(name: &str, check_caps: bool) -> Option<Display> {
                 let empty = "".to_string();
                 let merged = format!(
                     "{} {}",
```

Comment:
Thanks! Interesting, do you think we could get away with simply appending manufacturer ID to the end here, as the third argument? Just to avoid having logic specific for some null byte, maybe other people would also prefer to match on manufacturer ID for other reasons :thinking:

Reply:
oh, it can yes

Reply:
I'm thinking to keep it easy without additional separator, unless there was a reason you wanted to have a dash? Otherwise happy to merge if it helps you with your use-case!

Reply:
lol, i put because `wl-randr` does it, my bad

```bash
> wlr-randr
HDMI-A-1 "Invalid Vendor Codename - RTK 0x0101 0x01010101 (HDMI-A-1)"
  Make: Invalid Vendor Codename - RTK
  Model: 0x0101
  Serial: 0x01010101
```

Reply:
Thank you! :pray: